src/lesson/lesson03.js

```diff
@@ -1174,51 +1271,48 @@ function Lesson03() {
       ),
       "",  // initialCode
       Constants.Lesson03.SUCCESS_MESSAGE,
-      20
+      25
     )
   );
 
-  // Step 14
+  commandsReference = [commandsReference,
+                       Constants.References.MACHINE_SENSOR];
+
+  // Step 17
```

Comment:
In this step, it's possible to not use conditionals and brute-force the solution. I think you could set require_conditionals = true here.